aws/table_aws_rolesanywhere_profile.go

```diff
@@ -0,0 +1,178 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/rolesanywhere"
+
+	rolesanywherev1 "github.com/aws/aws-sdk-go/service/rolesanywhere"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsRolesAnywhereProfile(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_rolesanywhere_profile",
+		Description: "AWS Roles Anywhere Profile",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AnyColumn([]string{"id"}),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ResourceNotFoundException"}),
+			},
+			Hydrate: getProfile,
+			Tags:    map[string]string{"service": "rolesanywhere", "action": "GetProfile"},
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listProfiles,
+			Tags:    map[string]string{"service": "rolesanywhere", "action": "ListProfiles"},
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(rolesanywherev1.EndpointsID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "id",
+				Description: "The ID of the profile.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ProfileId"),
+			},
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) specifying the profile.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ProfileArn"),
+			},
+			{
+				Name:        "name",
+				Description: "The name of the profile.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "accept_role_session_name",
+				Description: "Accept custom role session names.",
```

Comment:
Could you please update the column descriptions to align with the API response descriptions? You can refer to the details here: [API_ProfileDetail](https://docs.aws.amazon.com/rolesanywhere/latest/APIReference/API_ProfileDetail.html).

Kindly update the descriptions for the remaining columns as well.